common/predictive-text/worker/index.ts

```diff
@@ -123,7 +131,10 @@ class LMLayerWorker {
     if(im.message == 'load') {
       let data = im as LoadMessage;
       if(data.model == this._currentModelSource) {
-        console.warn("Duplicate model load message detected - squashing!");
+        // Some JS implementations don't allow web workers access to the console.
+        if(console) {
```

Comment:
This is post-mortem. I got a KMW error `Uncaught ReferenceError: console is not defined`.
Should this be
```suggestion
        if(typeof console !== 'undefined') {
```

Reply:
Yes.  It was supposed to be in there; maybe I accidentally removed it when I had started to splinter the PR in two, then decided against it.

Reply:
Good catch @darcywong00 . Will you turn this into a PR?

Reply:
I've added it to wip #2389 and will park it for now.